vimeo/VimeoNetworking
# Moves AFNetworking files into VimeoNetworking

#### Ticket

N/A

#### Pull Request Checklist


#### Issue Summary

We decided to fold all AFNetworking files into our own repo instead of managing them as an external dependency. This will allow us better control of file visibility and also what should/shouldn't be included.

The motivator for this change was an issue Magisto has started to encounter when consuming our library, where Apple is giving warnings for deprecated use of `UIWebView` which will soon end up in rejected submissions.

There is a PR on AFNetworking [here](https://github.com/AFNetworking/AFNetworking/issues/4428) that has been opened since late August and still waiting to be merged, which indicates the support for the library is slowing down substantially and we don't expect any major updates will be available that we may be able to take advantage of.

#### Implementation Summary

* Remove AFNetworking from dependency list on Podspec and example project Podfile
* Add current version (3.1.0) files to VimeoNetworking inside a `ThirdParty` folder
* Add imports to required files in `VimeoNetworking.h` umbrella header so the code is visible in Swift
* Remove all Cartahge/Checkout files that are no longer needed

#### How to Test

- Consume this branch from the main app target, ensure you can build and run, all tests pass and that the app works as expected.
